From f69d07977074654dacfea3b2e07b1c8735387d46 Mon Sep 17 00:00:00 2001
From: Taylor Viti <tviti@hawaii.edu>
Date: Fri, 29 Nov 2019 15:03:15 -1000
Subject: [PATCH] patch commit

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 9d143c4..431a3d7 100644
--- a/Makefile
+++ b/Makefile
@@ -52,13 +52,13 @@ debian:
 .PHONY: app-bundle
 app-bundle: next
 	mkdir -p ./Next.app/Contents/MacOS
-	mkdir -p ./Next.app/Resources
+	mkdir -p ./Next.app/Contents/Resources
 	mv ./next ./Next.app/Contents/MacOS
 	cp -r ./ports/pyqt-webengine/* ./Next.app/Contents/MacOS
 	mv ./Next.app/Contents/MacOS/next-pyqt-webengine.py ./Next.app/Contents/MacOS/next-pyqt-webengine
 	chmod +x ./Next.app/Contents/MacOS/next-pyqt-webengine
 	cp ./assets/Info.plist ./Next.app/Contents
-	cp ./assets/next.icns ./Next.app/Resource
+	cp ./assets/next.icns ./Next.app/Contents/Resources
 
 .PHONY: install-app-bundle
 install-app-bundle:
-- 
2.23.0

